### DISCUSSION TITLE
A Simple Rustemo Tutorial

### DISCUSSION THREAD

**Discussion by fogarecious** [igordejanovic/rustemo#14] [Show and tell]
Hi, I would like to share a [simple tutorial](https://github.com/fogarecious/rustemo_tutorial) that I have written for this project.

The tutorial is for absolute beginners. After reading the tutorial, developers should have no problems to proceed to read the official tutorial.

**Comment by igordejanovic** [comment]
Thanks @fogarecious! Looks good. I've linked the tutorial from the "Where to start?" section and Rustemo README.
